pkg/epinio/models/epinio-resource.js

```diff
@@ -49,6 +49,10 @@ export default class EpinioResource extends Resource {
     opt.method = 'delete';
 
     try {
+      if (this.type === 'namespaces') {
```

Comment:
We can override the `remove` method in the `namespace` model and once we've closed logs call `super.remove`